src/convert.jl

```diff
@@ -1178,3 +1178,24 @@ for T in [HPolygon, HPolygonOpt, HPolytope, HPolyhedron]
     end
 end
 end
+
+"""
+    convert(::Type{LazySets.STAR}, P::AbstractPolyhedron{N}) where {N}
+
+Converts a polyhedral set into a star set.
+
+### Input
+
+- `STAR` -- type used for dispatch
+- `P`    -- polyhedral set
+
+### Output
+
+A star set.
+"""
+function convert(::Type{LazySets.STAR}, P::AbstractPolyhedron{N}) where {N}
```

Comment:
```suggestion
function convert(::Type{STAR}, P::AbstractPolyhedron{N}) where {N}
```
i think internally to the function we don't need to write `LazySets.STAR` (same in the functions' docstring)